draft-turner-lamps-nist-pqc-kem-certificates.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: Algorithm Identifiers for NIST's PQC Algorithms for Use in the Internet X.509 Public Key Infrastructure
-abbrev: PQC KEM for Certificates
+title: Algorithm Identifiers for NIST's PQC KEM Kyber Algorithm for Use in the Internet X.509 Public Key Infrastructure
+abbrev: PQC Kyber for Certificates
 category: std
 
-docname: draft-turner-lamps-nist-pqc-kem-certificates-latest
+docname: draft-ietf-lamps-nist-pqc-kem-kyber-certificates-latest
```

Comment:
Get rid of "latest" here. 
I made this mistake in another draft of mine.

Reply:
ack